LCM/dsc/engine/ConfigurationManager/MSFT_DSCLocalConfigurationManager.c

```diff
@@ -22,8 +22,9 @@
 #include "EngineHelper.h"
 #include "EventWrapper.h"
 #include "Resources_LCM.h"
-
-
+#include <signal.h>
+// Variable to store old Signal for SIGCHLD
+struct sigaction oldAction ;
```

Comment:
Should probably be static (i.e. private to this *.c file)

Reply:
ya, that would make it more clear.